src/OnChain.ts

```diff
@@ -91,7 +92,13 @@ export const OnChainMixin = (superclass) =>
         const { lnd } = getActiveOnchainLnd()
 
         const sendTo = [{ address, tokens: amount ?? defaultAmount }]
-        ;({ fee } = await getChainFeeEstimate({ lnd, send_to: sendTo }))
+        try {
+          ;({ fee } = await getChainFeeEstimate({ lnd, send_to: sendTo }))
```

Comment:
do you know in what scenario this would typically happen?

Reply:
Nope, not sure. But I just saw loads of uncaught errors in our logs which was causing the pager duty alerts.
We can either have this as an error in which case we'll continue to have this counted in our pager duty alerts.
Or if we think it's a non-issue then we can keep it at warn.

Reply:
I think we want to try to understand why that could be the case. maybe amount is very low?

Reply:
Yep, I think it's because of a low amount. I tried a bunch of low amount queries to getFee and it increased those errors in our logs

Reply:
oh yeah if there were not the dust check, then this might be the reason